### ISSUE TITLE
need to move vsc.install to its own repo

### ISSUE THREAD

**Issue by stdweird** [hpcugent/vsc-base#194]
any objections? the repo could also hold wrppers for running the jenkins unittests, whihc are unreviewed and look funny

**Comment by stdweird** [comment]
@JensTimmerman @boegel toughts?

**Comment by boegel** [comment]
I don't see the big advantage of fleshing it out into a separate repo/pkg, introducing our own dependency in vsc-base just so we can install/test it is weird?

Adding scripts for the Jenkins unit tests here still makes sense though.

**Comment by stdweird** [comment]
here's a good reason: https://github.com/stdweird/vsc-base/commit/7cd32b5762a081fda027271418f1d3c673dbe52c (and some of it's implications you're probably not aware of)
the jenkins scripts should be part of vsc-base, should be the same script for anything that uses `vsc.install` (it's only a build dependency afterall)

**Comment by boegel** [comment]
OK, I guess that is a good reason. And indeed, since it's only a build dependency for vsc-base, it's OK-ish (and will lead to a cleaner vsc-base repo/install).

So: +1 on fleshing out `vsc-install`.

This does imply having both `vsc-install` and `vsc-base` on PyPI (too) btw, since this affects the EB bootstrap procedure too (but things should work just fine, no changes should be needed there).

**Comment by boegel** [comment]
@stdweird: What other implications are you referring to btw? Is there more going on that just making sure that `import vsc.install` works with that hack (which will no longer be needed if vsc-install is proper (build-only) dep).

**Comment by JensTimmerman** [comment]
how does this solve anything? Aren't you just moving the problem? 
vsc.install will still depend on vsc.install to install, and as such the hack will still be needed in it's `setup.py` file?

**Comment by stdweird** [comment]
yes, but only in that file and it will  be a build_requires for vsc-base (i mean it's ok to have the hack for vsc.install, but is a but to coarse for vsc-base)

**Comment by JensTimmerman** [comment]
surely it will also be `install_requires` ? and as such will still be pulled in during installation?

**Comment by stdweird** [comment]
only `build_reqiures`, why would you need `vsc.install` to run anything?

**Comment by JensTimmerman** [comment]
installation requires it, you need it to run `python setup.py install .` in vsc-base
